src/tabs/GetStarted.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Created by Lightning App Builder</description>
+    <flexiPage>GetStarted</flexiPage>
+    <label>Get Started with NPSP</label>
+    <motif>Custom83: Pencil</motif>
```

Comment:
@smarovich Do you have any preference on the tab icon for the new Get Started tab? A full list of all available icons can be found [here](https://www.lightningdesignsystem.com/icons/) -- scroll down to the section labeled "Custom Labels" for the ones that are available for custom tabs.

Reply:
I'll check with Ali.

Reply:
Please use custom96 (the signpost).